Mailbox/Mailbox.cs

```diff
@@ -1,4 +1,40 @@
-﻿namespace Mailbox
+﻿using System;
+
+namespace Mailbox
 {
+    public class Mailbox
+    {
+        public Size Size { get; set; }
+        public ValueTuple<int, int> Location { get; set; }
```

Comment:
If you use the simpler value tuple syntax you can given names to the items in the tuple rather than just using Item1 and Item2.
```suggestion
        public (int X, int Y) Location { get; set; }
```